test/Makefile

```
@@ -56,7 +56,7 @@ coqscript: basics.vo
                       >tmp_test.v \
 	     && (echo $$i '-> coqc' | tr -d '\n'; \
 	         coqc -I .. tmp_test.v); then \
-	    echo '
' | tr -d '\n'; \
+	    echo $$'\r' | tr -d '\n'; \
```

In modern Unix, I'd write it like this:
```suggestion
	    printf '\r'; \
```

---

Thank you for suggesting these changes, I will rebase to update these lines.